src/core/parse.js

```diff
@@ -8,52 +8,55 @@ export let parse = (obj, selector) => {
     let outer = '';
     let blocks = '';
     let current = '';
-    let next;
 
     for (let key in obj) {
         let val = obj[key];
 
-        // If this is a 'block'
-        if (typeof val == 'object') {
-            next = selector
-                ? // Go over the selector and replace the matching multiple selectors if any
-                  selector.replace(/([^,])+/g, (sel) => {
-                      // Return the current selector with the key matching multiple selectors if any
-                      return key.replace(/(^:.*)|([^,])+/g, (k) => {
-                          // If the current `k`(key) has a nested selector replace it
-                          if (/&/.test(k)) return k.replace(/&/g, sel);
-
-                          // If there's a current selector concat it
-                          return sel ? sel + ' ' + k : k;
-                      });
-                  })
-                : key;
-
-            // If these are the `@` rule
-            if (key[0] == '@') {
-                // Handling the `@font-face` where the
-                // block doesn't need the brackets wrapped
-                if (key[1] == 'f') {
-                    blocks += parse(val, key);
+        // If this isn't an empty rule
+        if (val != undefined && val != null) {
```

Comment:
I think we could just check for: `val != undefined` since `null` is also equal to `undefined` but not strictly equal. What do you think?
```suggestion
        if (val != undefined) {
```

Reply:
@cristianbote can we use
```suggestion
        if (val != null) {
```